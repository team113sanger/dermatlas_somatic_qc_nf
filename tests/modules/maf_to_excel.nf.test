nextflow_process {

    name "Test Process MAF_TO_EXCEL"
    script "modules/maf_to_excel.nf"
    process "MAF_TO_EXCEL"

    test("Should run without failures") {

        when {
            params {
                // define parameters here. Example:
                // outdir = "tests/results"
            }
            process {
            """
            input[0] = tuple([empty: 'test'], file("$baseDir/tests/testdata/example_maf.maf"))
            """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
