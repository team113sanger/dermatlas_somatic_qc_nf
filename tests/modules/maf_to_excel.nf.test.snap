{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "example.maf.xlsx:md5,6ddebbbe80e15f0ce1a195476b9a2106"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-07-27T15:43:01.034548"
    }
}